package.json

```diff
@@ -59,7 +60,7 @@
     "eslint-config-react-app": "2.1.0",
     "eslint-plugin-flowtype": "2.44.0",
     "eslint-plugin-import": "2.8.0",
-    "eslint-plugin-jsx-a11y": "6.0.3",
+    "eslint-plugin-jsx-a11y": "5.1.1",
```

Comment:
Why back to a previous version?

Reply:
eslint-config-react-app needs a 5.x version to work in all cases, my editor was showing a failure due to conflict on every file.